documentation/docs/adapters/MythicMobsAdapter/entries/action/spawn_mob.mdx

```diff
@@ -0,0 +1,29 @@
+import {ActionsField, EntryField} from "@site/src/components/EntryField";
+
+# Spawn Mob Action
+
+The `Spawn Mob Action` action spawn MythicMobs's mob to the world.
+
+## How could this be used?
+
+Mix it up with something else to bring out the Mob.
```

Comment:
How are you using this entry? You can describe similar functionality.

The `How can this be used?` section is meant to give new users ideas of how to use an entry. Give ideas of scenarios where this entry could be useful.